app/Notifications/Nova/DuesAreLive.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Notifications\Nova;
+
+use App\Models\User;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldBeUnique;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Laravel\Nova\Notifications\NovaNotification;
+use Laravel\Nova\URL;
+
+class DuesAreLive extends NovaNotification implements ShouldQueue, ShouldBeUnique
+{
+    use Queueable;
+
+    /**
+     * The icon used for the notification.
+     *
+     * @var string
+     */
+    public $icon = 'currency-dollar';
+
+    /**
+     * The message used for the notification.
+     *
+     * @var string|null
+     */
+    public $message = 'Dues are live! Take a moment to pay online now.';
+
+    /**
+     * The text used for the call-to-action button label.
+     *
+     * @var string
+     */
+    public $actionText = 'Get started';
```

Comment:
I'd prefer action-oriented text on this button as well, but this `Dues are live` notification type seems like overlaps a lot with the `Dues payment due` notification.  I almost feel like we should only show one or the other.

Reply:
`DuesAreLive` means you need to create a `DuesTransaction`, `DuesPaymentDue` means you need to complete a `Payment`. The prune job will delete a `DuesAreLive` notification once a `DuesTransaction` is created.

Reply:
That all makes sense now, might be good to leave a quick note with the intended distinction in a comment on one/both of these classes for posterity

Reply:
Added comments to both classes.